frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeVegetarian.vue

```diff
@@ -0,0 +1,788 @@
+          >
+            le cadre pour la mise en oeuvre du plan pluriannuel de diversification des sources de protéines
+          </a>
+        </p>
+      </v-card-text>
+      <v-divider></v-divider>
+
+      <v-card-text>
+        <v-form ref="form" v-model="formIsValid">
+          <!-- daily vegetarian option in place? -->
+          <v-checkbox v-model="expe.hasDailyVegetarianOffer">
+            <template v-slot:label>
+              <span class="body-2 grey--text text--darken-3">
+                J'ai déjà mis en place l'option végétarienne quotidienne
+              </span>
+            </template>
+          </v-checkbox>
+
+          <!-- daily vegetarian option implementation date -->
+          <label v-if="expe.hasDailyVegetarianOffer" class="body-2 grey--text text--darken-3" for="date">
+            Date de mise en place
+          </label>
+          <v-menu
+            v-if="expe.hasDailyVegetarianOffer"
+            v-model="vegetarianDailyStartDateMenu"
+            :close-on-content-click="true"
+            transition="scale-transition"
+            offset-y
+            min-width="auto"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-text-field
+                :value="humanReadableDate(expe.dailyVegetarianOfferStartDate)"
+                prepend-icon="mdi-calendar"
+                readonly
+                v-bind="attrs"
+                v-on="on"
+                hide-details="auto"
+                solo
+                dense
+                style="max-width: 200px"
+                id="date"
+                class="mt-2 mb-4"
+              ></v-text-field>
+            </template>
+
+            <v-date-picker
+              v-if="expe.hasDailyVegetarianOffer"
+              v-model="expe.dailyVegetarianOfferStartDate"
+              :max="today"
+              locale="fr-FR"
+            ></v-date-picker>
+          </v-menu>
+
+          <!-- launch date -->
+          <label class="body-2 grey--text text--darken-3" for="launch-date">
+            Date de lancement de l’expérimentation
+          </label>
+          <v-menu
+            v-model="launchDateMenu"
+            :close-on-content-click="true"
+            transition="scale-transition"
+            offset-y
+            min-width="auto"
+          >
+            <template v-slot:activator="{ on, attrs }">
+              <v-text-field
+                :value="humanReadableDate(expe.experimentationStartDate)"
+                prepend-icon="mdi-calendar"
+                readonly
+                v-bind="attrs"
+                v-on="on"
+                hide-details="auto"
+                dense
+                solo
+                style="max-width: 200px"
+                id="date"
+                class="mt-2 mb-4"
+              ></v-text-field>
+            </template>
+
+            <v-date-picker v-model="expe.experimentationStartDate" :max="today" locale="fr-FR"></v-date-picker>
```

Comment:
Should this date be able to be in the future, in case they have an experimentation planned to start?

Reply:
:thinking: Yeah let's do that, I'd rather have it be too lax than unnecessarily complicated. Fix here : https://github.com/betagouv/ma-cantine/pull/1237/commits/597f6a0ea8483a4b04c7d4f743d9fdf7705f299c